docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
     Use "dce [command] --help" for more information about a command.
     ```
 
-4. Type `dce init` to configure dce via an interactive prompt. This will generate a config file at ~/.dce.yaml
+1. Type `dce init` to generate a new config file at ~/.dce.yaml. Leave everything blank for now.
+
+## Deploying DCE
+
+1. [Download and install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+1. Choose an AWS account to be your new "DCE Master Account" and configure the AWS CLI with user credentials that have AdministratorAccess in that account.
+
+    ```
+    ➜  ~ aws configure set aws_access_key_id default_access_key
+    ➜  ~ aws configure set aws_secret_access_key default_secret_key
+    ```
+
+1. Type `dce system deploy` to deploy dce to the AWS account specified in the previous step.
+
+1. Edit your dce config file with the host and base url from the api gateway that was just deployed to your master account. This can be found in the master account under `API Gateway > (The API with "dce" in the title) > Stages > "Invoke URL: https://<host>/<baseurl>"`. Your config file should look something like this:
+
+    ```
+    api:
+      host: abcdefghij.execute-api.us-east-1.amazonaws.com
+      basepath: /api
+    region: us-east-1
+    ```
+   
+## Authenticating with the DCE System
+
+DCE uses AWS Cognito to manager authentication and authorization.
+
+1. Open the AWS Console in your DCE Master Account and Navigate to AWS Cognito by typing `Cognito` in the search bar
```

Comment:
Might be worth a little intro here, explaining what we're doing
(ie. that we're manually setting up a user pool, though you can also federate with third party IDPs)

Also maybe a note like:

> We'll walk through setting this up in the AWS web console, but note that all of these operations can be automated using the AWS Cognito CLI or SDKs